examples/tiny-cluster-ingress.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: "druid.apache.org/v1alpha1"
```

Comment:
I  think we will end up with  a long list of confusing tiny-cluster specs  when adding one new file for each feature addition. I think adding documentation is probably better way to go at this point. so let us focus on  that instead of new spec . I think it  is mentioned in another ticket.

Reply:
sure ill remove the spec